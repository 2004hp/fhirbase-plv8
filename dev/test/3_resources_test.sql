--db:fhirb
--{{{

select * from fhir.resource_indexables
where resource_type = 'Patient' ;

--}}}
