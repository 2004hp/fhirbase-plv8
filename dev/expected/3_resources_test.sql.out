select count(*) from fhir.resource_indexables
where resource_type = 'Patient' ;
 count
-------
    16
(1 row)

